modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -21,189 +40,144 @@ UnitTest.asynctest('browser.tinymce.core.content.EditorContentTest', (success, f
     return body;
   };
 
-  Theme();
-
-  const toHtml = (node: AstNode) => {
+  const toHtml = (node: AstNode): string => {
     const htmlSerializer = HtmlSerializer({});
     return htmlSerializer.serialize(node);
   };
 
-  let events = [];
-  const logEvent = (event: EditorEvent<{}>) => {
-    events.push(event.type);
+  const assertEventsFiredInOrder = (expectedEvents: string[] = defaultExpectedEvents) => {
+    assert.deepEqual(events, expectedEvents, 'Get content events should have been fired');
   };
+  const clearEvents = () => events = [];
 
-  const defaultExpectedEvents = [
-    'beforesetcontent',
-    'setcontent',
-    'beforegetcontent',
-    'getcontent'
-  ];
-  const sAssertEventsFiredInOrder = (expectedEvents: string[] = defaultExpectedEvents) => Step.sync(() => {
-    Assertions.assertEq('Get content events should have been fired', expectedEvents, events);
+  beforeEach(() => clearEvents());
+
+  it('TBA: getContent html', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>html</p>');
+    const html = editor.getContent();
+    Assertions.assertHtml('Should be expected html', '<p>html</p>', html);
+    assertEventsFiredInOrder();
+  });
+
+  it('TINY-6281: getContent html with empty editor', () => {
+    const editor = hook.editor();
+    editor.setContent('');
+    const html = editor.getContent();
+    Assertions.assertHtml('Should be expected html', '', html);
```

Comment:
This might not show up when searching for places to put in `TinyAssertions.assertContent()` so just noting it